test/test_stacked_bar.rb

```diff
@@ -9,50 +9,66 @@ def setup
       [:Charles, [80, 54, 67, 54]],
       [:Julie, [22, 29, 35, 38]]
     ]
-    @sample_labels = {
-      0 => '5/6',
-      1 => '5/15',
-      2 => '5/24'
-    }
   end
 
   def test_bar_graph
-    g = Gruff::StackedBar.new
+    @datasets = [
+      [:Jimmy, [25, 36, 86, 39]],
+      [:Charles, [80, 54, 67, 54]],
+      [:Julie, [2, 29, 35, 38]]
```

Comment:
I don't know that having Julie start with 2 instead of 22 was intentional.  This could be update to use the default data like the other tests.  The test passes with this data.